src/renderers/testing/ReactTestRenderer.js

```diff
@@ -60,12 +62,12 @@ ReactTestComponent.prototype.receiveComponent = function(
   this._currentElement = nextElement;
   this.updateChildren(nextElement.props.children, transaction, context);
 };
+
 ReactTestComponent.prototype.getHostNode = function() {};
-ReactTestComponent.prototype.getPublicInstance = function() {
-  // I can't say this makes a ton of sense but it seems better than throwing.
-  // Maybe we'll revise later if someone has a good use case.
-  return null;
+ReactTestComponent.prototype.getPublicInstance = function(transaction) {
```

Comment:
Let's keep it the way it is. If we want to make this cleaner we should just use ES6 classes instead but this should be done separately. We should not mix stylistic and meaningful changes in the same PR.